src/menu/menu.js

```diff
@@ -27,20 +31,51 @@ export function getRestaurantMenu() {
         if (!Array.isArray(cart)) {
           cart = [];
         }
-        cart.push(item);
+
+        //get id from URL
+        const id = window.location.hash.split("/")[1];
+        console.log("the id is ", id);
+
+        const menuItem = { item, id };
+        cart.push(menuItem);
         localStorage.setItem("cart", JSON.stringify(cart));
 
+        totalQuantity++;
+        totalPrice += item.price;
+
         alert(`${item.name} has been added to your cart!`);
+        console.log(menuItem);
 
-        console.log(item);
+        // Update total row
+        totalQuantityCol.innerText = `Total quantity: ${totalQuantity}`;
+        totalPriceCol.innerText = `Total price: $${totalPrice.toFixed(2)}`;
       });
-
       row.appendChild(name);
       row.appendChild(topping);
       row.appendChild(price);
       row.appendChild(addToCartBtn);
-
       menuBody.appendChild(row);
     });
+
+    // Create total row
```

Comment:
This entire section for Total Price can be added in a separate function outside `getRestaurantMenu`.

This entire function `getRestaurantMenu` should be split in separate functions to clean it up where it makes sense so it's easier to read.